pynest/examples/EI_clustered_network/helper.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+#
+# helper.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""PyNEST EI-clustered network: Helper Functions
+-------------------------------------------
+
+Helper functions to calculate synaptic weights and first passage times for exponential synapses to construct
+random balanced networks.
+"""
+
+import numpy as np
+
+small = 1e-10
+
+
+def max_psp_exp(tau_m, tau_syn, c_m=1.0, e_l=0.0):
+    """
+    Calculates the maximum psp amplitude for exponential synapses and unit J.
```

Comment:
As a general rule we use [NumPy style docstrings](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html) - please modify the docstrings so they remain consistent with the rest of our code

Reply:
I did so in commit [3a3f444](https://github.com/nest/nest-simulator/pull/3028/commits/3a3f444f56179316ba17e892f2936333293ea5df). My local install of flake8 is now complaining about the docstrings. I was not able to find the right configuration and hope it still runs in the check correctly.